Documentation update

marcellorinaldo opened the pull request:
Signed-off-by: Marcello Martina <martina.marcello.rinaldo@outlook.com>

Brief description of the PR. Documentation updates

**Related Issue:** The actual documentation was not up-to-date. Some steps were wrong and some images didn't correspond to the newest version of kura.

**Description of the solution adopted:** Walkthrough of the documentation to find errors. Updates made to the GitHub pages markdown files.

**Screenshots:** N/A.

**Any side note on the changes made:** N/A.

marcellorinaldo committed:
Documentation update

Signed-off-by: Marcello Martina <martina.marcello.rinaldo@outlook.com>

pierantoniomerlino commented on _posts/config/2017-05-04-device-information.md:
```diff
@@ -21,10 +21,12 @@ The **Threads** tab shows a list of the threads that are currently running in th
 
 ![]({{ site.baseurl }}/assets/images/config/DeviceSystemPackages.png)
+--->
 
```
This one is for a Kura 5 feature.

pierantoniomerlino commented on _posts/config/2017-05-04-device-information.md:
```diff
@@ -34,7 +36,10 @@ The **System Properties** tab shows a list of relevant properties including OS a
\ No newline at end of file
+![]({{ site.baseurl }}/assets/images/config/DeviceSystemLogs.png)
+
+--->
```
This one is for a Kura 5 feature.

pierantoniomerlino commented on _posts/config/2017-06-10-threat-manager.md:
```diff
@@ -1,44 +0,0 @@
-The following parameters are available:
-
-- *flooding.protection.enabled* : enables the application of the firewall rules for flooding protection.
-
```
This one is for a Kura 5 feature.

pierantoniomerlino commented on _posts/intro/2014-02-01-raspberry-pi-quick-start.md:
```diff
@@ -91,16 +91,18 @@ following steps:
-     Once trusted the source, the user will be redirected to a login page where the default **username** is:
+     <!---Once trusted the source, the user will be redirected to a login page where the default **username** is:--->
+     
+     The user will be prompted for login credentials where the default **username** is:
```
This one is for a Kura 5 feature.

marcellorinaldo committed:
Update 2017-05-04-device-information.md

Added System Packages and System Logs sections since they are present in Kura 5

marcellorinaldo committed:
Update 2014-02-01-raspberry-pi-quick-start.md

Https functionalities are present on Kura 5

marcellorinaldo committed:
Restored Threat Management section

Signed-off-by: Marcello Martina <martina.marcello.rinaldo@outlook.com>

MMaiero commented:
Since we don't have the capability to separate the docs per version, maybe we can add a note on what is for Kura 5 and what is for Kura 4.
